packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -64,11 +72,16 @@ const CenterWrapper = styled.div`
 
   a {
     color: ${C_CONSENT_ACTION};
-    text-decoration: underline;
-    text-decoration-color: ${C_PEBBLE};
+    text-decoration: none;
+    border-bottom: solid 0.0625rem ${C_PEBBLE};
 
     ${hoverFocusStyles}
   }
+
+  a:hover,
+  a:focus {
+    border-bottom: solid 0.125rem transparent;
```

Comment:
4. "When I hover over link elements
Then they have a visible state change"
"When I navigate through [component] by pressing the TAB key
Then all links have a visible state change
And this is in addition to the browser default focus outline styling" 👍